api/src/services/tickets/tickets.js

```diff
@@ -4,10 +4,6 @@ import * as util from 'src/lib/util'
 import { logger } from 'src/lib/logger'
 import { matrix } from 'src/lib/roles'
 import rules from 'src/rules/tickets/**.{js,ts}'
-let beforeRulesArr = util.loadRules(rules, "before")
-let afterRulesArr = util.loadRules(rules, "after")
-
-//requireAuth({ role: READ_TASK_ROLES })
 
```

Comment:
Removed top level rules load as we now need to also specify the "type" of rules we want to load in.